monitors/didyoumean.js

```diff
@@ -1,24 +1,38 @@
-const levenshtein = require("fast-levenshtein");
+/* eslint-disable no-use-before-define, no-restricted-syntax, consistent-return, guard-for-in */
+
+let levenshtein;
+
+exports.minDist = null;
+
+exports.init = (client) => {
+  levenshtein = require("fast-levenshtein");
```

Comment:
????? This is Komada 0.20.x, not 0.18.1, the download command supports packages.

Reply:
It didn't when i was testing it initially.

Kept firing `cannot find module fast-levenshtein`

Reply:
probs just a weird issue with my docker. anyways reverted that line to be outside init